src/parameters.toml

```
@@ -491,6 +491,222 @@ alias = "k_thrshld_stpnss"
+
+[mam3_nucleation_a_i_ion_induced]
+alias = "a_i"
+value = 0.00483140
+type = "float"
+description = "Parameter for pure ternary H2SO4-NH3-H2O nucleation. From Dunne et al., 2016. DOI: 10.1126/science.aaf2649."
+
+[mam3_nucleation_a_1_neutral]
+alias = "a_1"
+value = 0.0400097
+type = "float"
+description = "Parameter for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_a_2_neutral]
+alias = "a_2"
+value = 1.84826
+type = "float"
+description = "Parameter for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_a_3_ion_induced]
+alias = "a_3"
+value = 0.00136641
+type = "float"
+description = "Parameter for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_a_4_ion_induced]
+alias = "a_4"
+value = 1.56588
+type = "float"
+description = "Parameter for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_a_5]
+alias = "a_5"
+value = 0.186303
+type = "float"
+description = "Parameter for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_Y_MTO3_percent]
+alias = "Y_MTO3"
+value = 0.029
+type = "float"
+description = "Highly oxygenated organic molecule (HOM) percent yield from monoterpene (MT) oxidation with ozone, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_Y_MTOH_percent]
+alias = "Y_MTOH"
+value = 0.012
+type = "float"
+description = "Highly oxygenated organic molecule (HOM) percent yield from monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_MTO3_organic_factor]
+alias = "k_MTO3"
+value =  8.05e-16
+type = "float"
+description = "Factor for temperature-dependent rate of monoterpene (MT) oxidation with ozone, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_MTOH_organic_factor]
+alias = "k_MTOH"
+value =  1.2e-11
+type = "float"
+description = "Factor for temperature-dependent rate of monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_exp_MTO3_organic_factor]
+alias = "exp_MTO3"
+value = -640
+type = "float"
+description = "Additional factor for monoterpene (MT) oxidation with ozone, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_exp_MTOH_organic_factor]
+alias = "exp_MTOH"
+value =  440
+type = "float"
+description = "Additional factor for monoterpene (MT) oxidation with hydroxyl radicals, used for pure organic nucleation. From Kirkby et al., 2016. DOI: 10.1038/nature17953."
+
+[mam3_nucleation_k_H2SO4org_factor]
+alias = "k_H2SO4org"
+value =  3.27e-15
```

```suggestion
value =  3.27e-21
```
Since we will do the SI conversions on the CM.jl side for now